cli/internal/config/config.go

```diff
@@ -0,0 +1,89 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"slices"
+	"strings"
+
+	"github.com/adrg/xdg"
+)
+
+const configPath = "cloudquery/config.json"
+
+var configKeys = []string{
+	"team",
+}
+
+// GetValue reads the value of a config key from the config file
+func GetValue(key string) (string, error) {
+	if !slices.Contains(configKeys, key) {
+		return "", fmt.Errorf("invalid config key %v (options are: %v)", key, strings.Join(configKeys, ", "))
+	}
+	configFilePath, err := xdg.ConfigFile(configPath)
+	if err != nil {
+		return "", fmt.Errorf("failed to get config file path: %w", err)
+	}
+	b, err := os.ReadFile(configFilePath)
```

Comment:
Can we verify that reading a value from the config doesn't fail if the file doesn't exist? For example if I run `cloudquery switch` without arguments and before setting any team seems like we fail with an error now.

I would expect it to print the help instead if there's no team

Reply:
Good point, will add a test for this and fix if needed

Reply:
Added a test for this and fixed in [75ec8e6](https://github.com/cloudquery/cloudquery/pull/14493/commits/75ec8e68422b85ffadcffa724ef1d20a24effa92)